### DISCUSSION TITLE
Using OMD behind a reverse proxy

### DISCUSSION THREAD

**Discussion by MBoutray** [open-metadata/OpenMetadata#18479] [Q&A]
Hello,
I'm trying to make a set of services accessible at their own prefixed path (example.com/openmetadata for example) using a reverse proxy. I'm having trouble with OMD, I can't seem to find a documentation or working examples on how to do it. Are the x-forwarded headers correctly handled in the code? (all my services are dockerized)

Thanks in advance

**Comment by Prajwal214** [comment]
Refer the below docs
https://docs.open-metadata.org/v1.4.x/deployment/docker#run-openmetadata-with-a-load-balancer

**Reply by MBoutray** [reply]
Thank you for your answer, I have already read this section of the documentation, but it doesn't mention if the app can handle prefixed requests. I'm sorry if my question isn't really clear, those notions are pretty new to me.

**Comment by jojo599k** [comment]
Looking for this?
https://docs.open-metadata.org/latest/deployment/docker/subpath

**Reply by MBoutray** [reply]
Hi,
It looks very promising indeed.
I'll try it out asap, and keep you informed on the outcome.
Have a good day.
